Plutarch/Api/V1/Tx.hs

```diff
@@ -90,7 +90,7 @@ newtype PTxOut (s :: S)
           s
           ( PDataRecord
               '[ "address" ':= PAddress
-               , "value" ':= PValue
+               , "value" ':= PValue 'Sorted 'NonZero
```

Comment:
This should be `Positive`, but you may want to add a warning wrt. the babel stuff, because then you can have negative value in the output.